src/main/java/org/jfrog/hudson/generic/ArtifactoryGenericConfigurator.java

```diff
@@ -221,15 +245,23 @@ public boolean isAsyncBuildRetention() {
     }
 
     public boolean isEnableIssueTrackerIntegration() {
-        return false;
+//        return false;
+        return enableIssueTrackerIntegration;
+    }
+
+    public boolean isFilterExcludedArtifactsFromBuild() {
+        return filterExcludedArtifactsFromBuild;
     }
 
     public boolean isAggregateBuildIssues() {
-        return false;
+//        return false; #(6 years ago since 2019) Shay Yaakov: HAP-320 - Issue tracking doesn't work in Maven2
+        return aggregateBuildIssues;    //We don't care about Maven2 anymore in 2019, do we?
     }
 
     public String getAggregationBuildStatus() {
-        return null;
+ //       return null; #(6 years ago since 2019) Shay Yaakov: HAP-320 - Issue tracking doesn't work in Maven2
+        return aggregationBuildStatus;  //We don't care about Maven2 anymore in 2019, do we?
+
```

Comment:
The above code should probably be cleaned up.

Reply:
No problem, will do. I was wondering if it is a good way to explain why the method now is going to change the return value.

Reply:
Cleaned up.